package.json

```diff
@@ -41,14 +43,21 @@
     "preact": "^10.0.4",
     "purify-css": "^1.2.5",
     "purifycss-webpack": "^0.7.0",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
```

Comment:
What's the reason for adding React?
The project uses Preact/compat.

Reply:
I see would be removed

Reply:
fixed with: mkkeck@f834bb0